test/run.sh

```diff
@@ -251,6 +251,15 @@ do
 
   case $ext in
   "mo")
+    if grep -q "//INCREMENTAL-GC-ONLY" $base.mo
```

Comment:
What was the reason for this tweak? Is it not ok to run the test on all gcs?

Reply:
I only used this for the `gc-trigger.mo` test. The test does not work for the generational GC, as it reclaims the garbage immediately, which is nice but then the GC trigger has no effect... Maybe, there is a better way to test the GC trigger, that it reclaims memory in all GC cases.